code/go/0chain.net/smartcontract/storagesc/allocation.go

```diff
@@ -1302,6 +1350,10 @@ func (sc *StorageSmartContract) canceledPassRates(alloc *StorageAllocation,
 			//	continue // already accepted, already rewarded/penalized
 			//}
 			var expire = oc.CreatedAt + toSeconds(getMaxChallengeCompletionTime())
+			if expire < oc.CreatedAt || expire < toSeconds(getMaxChallengeCompletionTime()) {
+				return nil, fmt.Errorf("overflow while calculating expire time")
```

Comment:
We don't need to worry about the time overflow in seconds, check https://stackoverflow.com/questions/25065055/what-is-the-maximum-time-time-in-go